labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -70,26 +64,22 @@ object ListManipulationExercise01 {
    * - ... whichever way you like 
    * 
    */
-  def sortList[T <% Ordered[T]](list: List[T]): List[T] = {
-    error("fix me")
-  }
+  def sortList[T <% Ordered[T]](list: List[T]): List[T] = list.sorted
 
   /**
    * Check whether a given element in a list exists, i.e. elementExists(List("a", "b", "c"), "b") = true
    * Again, easy to implement using built-in functionality, but also possible to implement in your own free-style way.
    */
-  def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
-  }
+  def elementExists[T](l: List[T], e: T): Boolean = l.foldLeft(false)((x,i) => if (x)
+    {x} else {if (i == e) true else false}) 
+      
```

Comment:
If statements that return booleans are a good target for rewriting. For example here, you could use `i == e`. After that, the outermost if statement could become `x || (i == e)`. Of course, since Scala has everything under the sun built in, folding booleans using "or" is ".exists(predicate)", and folding them using "and" is ".forAll(predicate)".